<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="clientes_movimento_VSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QFrame" name="clientesFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="clientesLabel">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 127);
font: bold 10pt &quot;Verdana&quot;;</string>
         </property>
         <property name="text">
          <string>Clientes: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="clientesTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="clientesButtonsHLayout">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="clienteDeleteBtn">
           <property name="text">
            <string>Excluir</string>
           </property>
           <property name="icon">
            <iconset resource="recursos/recursos.qrc">
             <normaloff>:/imagens/delete.png</normaloff>:/imagens/delete.png</iconset>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="clientesButtonsHSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="movimentoFrame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QLabel" name="movimentoLabel">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 127);
font: bold 10pt &quot;Verdana&quot;;</string>
         </property>
         <property name="text">
          <string>Movimentos: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="movimentoTable"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="movimentoButtonsHLayout">
         <item>
          <widget class="QGroupBox" name="movimentoEditGroup">
           <property name="title">
            <string> Edição: </string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="movimentoEditBtn">
              <property name="text">
               <string>Editar</string>
              </property>
              <property name="icon">
               <iconset resource="recursos/recursos.qrc">
                <normaloff>:/imagens/update.png</normaloff>:/imagens/update.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="movimentoInsertBtn">
              <property name="text">
               <string>Inserir</string>
              </property>
              <property name="icon">
               <iconset resource="recursos/recursos.qrc">
                <normaloff>:/imagens/insert.png</normaloff>:/imagens/insert.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="movimentoDeleteBtn">
              <property name="text">
               <string>Excluir</string>
              </property>
              <property name="icon">
               <iconset resource="recursos/recursos.qrc">
                <normaloff>:/imagens/delete.png</normaloff>:/imagens/delete.png</iconset>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="movimentoNavGroup">
           <property name="title">
            <string>Navegação: </string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>5</number>
            </property>
            <property name="topMargin">
             <number>3</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="movimentoNavFirstBtn">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos/recursos.qrc">
                <normaloff>:/imagens/first.png</normaloff>:/imagens/first.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="movimentoNavPrevBtn">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos/recursos.qrc">
                <normaloff>:/imagens/prev.png</normaloff>:/imagens/prev.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="movimentoNavNextBtn">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos/recursos.qrc">
                <normaloff>:/imagens/next.png</normaloff>:/imagens/next.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="movimentoNavLastBtn">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="recursos/recursos.qrc">
                <normaloff>:/imagens/last.png</normaloff>:/imagens/last.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="movimentoButtonsHSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>28</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recursos/recursos.qrc"/>
 </resources>
 <connections/>
</ui>
